package ovh.cerebrum.cerebrumapp.exception;

public class AlreadyExistException extends Exception {
}
